xml/vt_introduction.xml

```diff
@@ -195,12 +195,9 @@
    </listitem>
    <listitem>
     <para>
-     Paravirtual mode does not require the host computer to support
-     <xref linkend="gloss.vt.hwassisted"/> virtualization technology, but
-     does require the guest operating system to be modified for the
-     virtualization environment. Typically, operating systems running in
-     paravirtual mode enjoy better performance than those requiring full
-     virtualization mode.
+     Paravirtual mode usually modifies guest operating system to suit better
+     in virtualization environment. Typically, operating systems running in 
+     paravirtual mode enjoy performance more than other stuff.
```

Comment:
Why did you rewrite this sentence?
"than other stuff" seems much harder to understand and much less specific.